### DISCUSSION TITLE
custom paper size & color

### DISCUSSION THREAD

**Discussion by azc666** [spatie/laravel-pdf#18] [Q&A] [answered]
Does this package support custom paper sizes? Can I add one in Enum?
Does this package support spot color creation or is it possible to convert a color to spot with js on the pdf?

**Comment by jeffreyvanhees** [comment]
You can access the browsershot instance like this:
```php
return Pdf::view('pdf.receipt-test')
  ->withBrowsershot(fn(Browsershot $browsershot) => $browsershot->paperSize(
     width: 1000, height: 1000,
));
```

**Reply by freekmurze** [reply]
I wouldn't mind a PR that adds a `paperSize` method on the `PdfBuilder`

**Reply by jeffreyvanhees** [reply]
> I wouldn't mind a PR that adds a `paperSize` method on the `PdfBuilder`

I'll take a look if I have time @freekmurze 👌🏻

**Comment by jeffreyvanhees** [comment, marked as answer]
@azc666 See https://github.com/spatie/laravel-pdf/pull/33

**Reply by azc666** [reply]
👍🏻👍🏻
